src/openrct2/scenario/Scenario.h

```diff
@@ -423,7 +423,7 @@ enum
 #define AUTOSAVE_PAUSE 0
 #define DEFAULT_NUM_AUTOSAVES_TO_KEEP 10
 
-static constexpr money32 COMPANY_VALUE_ON_FAILED_OBJECTIVE = 0x80000001;
+static constexpr money64 COMPANY_VALUE_ON_FAILED_OBJECTIVE = 0x8000000000000001;
```

Comment:
It needs to be checked if old saves that failed the objective are imported correctly.